hooks/pre-push.d/rs-spdx-license-test

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+files_without_license=$(find . -path ./target -prune -o -type f -name "*.rs" -exec grep -L "// SPDX-License-Identifier: Apache-2.0" {} \;)
+[ -z "$files_without_license" ] && exit 0
+printf "SPDX license ID missing from the following .rs files:\n$files_without_license\n" && exit 1
```

Comment:
1. We want to enforce that the header is the first line of the file.
2. The error message is incorrect if the file contains a header but not an `Apache-2.0` license.
3. I think the find invocation can be cleaner.

Reply:
This still needs to be done.